charts/ocis/templates/sharing/deployment.yaml

```diff
@@ -62,31 +62,31 @@ spec:
 
             # user sharing
             - name: SHARING_USER_DRIVER
-              value: cs3
-            - name: SHARING_USER_CS3_PROVIDER_ADDR
+              value: jsoncs3
+            - name: SHARING_USER_JSONCS3_PROVIDER_ADDR
               value: storage-system:9215
-            - name: SHARING_USER_CS3_SYSTEM_USER_API_KEY
+            - name: SHARING_USER_JSONCS3_SYSTEM_USER_API_KEY
               valueFrom:
                 secretKeyRef:
                   name: {{ .Values.secretRefs.storageSystemSecretRef }}
                   key: api-key
-            - name: SHARING_USER_CS3_SYSTEM_USER_ID
+            - name: SHARING_USER_JSONCS3_SYSTEM_USER_ID
               valueFrom:
                 secretKeyRef:
                   name: {{ .Values.secretRefs.storageSystemSecretRef }}
                   key: user-id
 
             # public sharing
             - name: SHARING_PUBLIC_DRIVER
-              value: cs3
-            - name: SHARING_PUBLIC_CS3_PROVIDER_ADDR
+              value: jsoncs3
+            - name: SHARING_PUBLIC_JSONCS3_PROVIDER_ADDR
```

Comment:
This does not work for me. There seems to be a bug in ocis when createing the reva config for the publicshare service. Will open an issue for that. Somehow the jsoncs3 backend only works when I set the `_CS3_` options as well. Need to take a deeper look.

Reply:
Ah ok, that makes sense.